.github/workflows/integration.yml

```
@@ -106,6 +106,7 @@ jobs:
     strategy:
       matrix:
         rust: [ stable ]
+        pyarrow: [ "13", "14" ]
```

👍 
We might also want to leave a comment here so we remember why we test 13 specifically.

```suggestion
        # PyArrow 13 was the last version prior to introduction to Arrow PyCapsules
        pyarrow: [ "13", "14" ]
```